content/applications/productivity/discuss/overview/plan_activities.rst

```diff
@@ -1,64 +1,194 @@
+Activities --> Activity Types`.
 
-.. image:: plan_activities/settings_activities_types.png
+.. image:: plan_activities/settings-activities-types.png
    :align: center
-   :height: 300
-   :alt: View of the settings page emphasizing the menu activity types for Odoo Discuss
+   :alt: View of the settings page emphasizing the menu activity types.
 
-.. tip::
-   If you need to create an activity type with an available calendar, make sure to create
-   it with an *Action to Perform* set as *Meeting*.
+To edit an existing activity type, select it from the list, then click :guilabel:`Edit`. To create a
+new activity type, click :guilabel:`Create`.
+
+Choose a :guilabel:`Name` for the new activity type.
 
-Recommend next activities
--------------------------
+.. image:: plan_activities/new-activity-type.png
+   :align: center
+   :alt: New activity type form.
+
+Activity settings
+-----------------
+
+Action
+~~~~~~
+
+*Actions* specify the intent of the activity. Some actions trigger specific behavior after an
+activity is scheduled.
+
+- If either **Phonecall** or **Meeting** are selected, users will have the option to open their
+  calendar to schedule a time for this activity.
+- If **Upload Document** is selected, a link to upload a document will be added directly to the
+  planned activity in the chatter.
+- If **Request Signature** is selected, a link will be added to the planned activity in the chatter
+  that will open a signature request pop-up.
+
+.. note::
+   The actions available to select on an activity type will vary depending on the applications that
+   are currently installed in the database.
+
+Default user
+~~~~~~~~~~~~
+
+To automatically assign this activity to a specific user when this activity type is scheduled,
+choose a name from the :guilabel:`Default User` drop-down. If this field is left blank, the activity
+will be assigned to the user who creates the activity.
+
+Default summary
+~~~~~~~~~~~~~~~
+
+To include any notes every time this activity type is created, enter them into the
+:guilabel:`Default Summary` field.
+
+.. note::
+   The information in the :guilabel:`Default User` and :guilabel:`Default Summary` fields will be
+   included when an activity is created. However, they can be altered before the activity is
+   scheduled or saved.
 
-Odoo helps you to plan a flow of activities by allowing you to set *Recommended Next Activities*.
+Next activity
+-------------
 
-.. image:: plan_activities/recommended_activities.png
+To automatically suggest or trigger a new activity after an activity has been marked complete, the
+*chaining type* must be set.
+
+Suggest next activity
+~~~~~~~~~~~~~~~~~~~~~
+
+In the :guilabel:`Chaining Type` field, select :guilabel:`Suggest Next Activity`. The field
+underneath will now be titled :guilabel:`Suggest`. Select any activities to recommend as follow-up
+tasks to this activity type.
+
```

Comment:
Would be beneficial to include why they may want to utilize the 'Email Templates' field beneath the 'Suggest' field before jumping into 'Schedule'